### PR TITLE
Lt 959 audit

### PR DESCRIPTION
**Pull request by jfaracik**

### CODE REVIEW

File: gov_users/views.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    Get user from pk
+    """
+    def get(self, request, pk):
+        gov_user = get_gov_user_by_pk(pk)
+
+        serializer = GovUserSerializer(gov_user)
+        return JsonResponse(data={'user': serializer.data},
+                            safe=False)
+
+    def put(self, request, pk):
```
Comment (lukesa) [inline gov_users/views.py]: add docstring and a swagger decorator

File: organisations/views/views_site.py [hunk 2]
```diff
@@ -79,6 +81,7 @@ def post(self, request, org_pk):
 
 
 class OrgSiteDetail(APIView):
+    authentication_classes = (GovAuthentication,)
     """
```
Comment (lukesa) [inline organisations/views/views_site.py]: move to 1st line

File: queues/views.py [hunk 3]
```diff
@@ -3,12 +3,14 @@
...
 
+from conf.authentication import GovAuthentication
 from queues.models import Queue
 from queues.serializers import QueueSerializer
 
 
 @permission_classes((permissions.AllowAny,))
 class QueuesList(APIView):
+    authentication_classes = (GovAuthentication,)
     """
```
Comment (lukesa) [inline queues/views.py]: move docstring to 1st line

File: teams/tests/tests_list_user_by_team.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+class UserByTeamListTests(DataTestClient):
+
+    def setUp(self):
+        super().setUp()
+
+    def tests_user_by_team_list(self):
+        team = Team.objects.get(name='Admin')
+        team2 = Team(name='Second')
+        team2.save()
+        govuser1 = GovUser(email='test2@mail.com', first_name='John', last_name='Smith', team=self.team)
```
Comment (lukesa) [inline teams/tests/tests_list_user_by_team.py]: common object

File: teams/tests/tests_list_user_by_team.py [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+        govuser2 = GovUser(email='test3@mail.com', first_name='John', last_name='Smith', team=team2)
+        govuser1.save()
+        govuser2.save()
+        url = reverse('teams:team_users', kwargs={'pk': team.id})
+        response = self.client.get(url, **self.gov_headers)
+        response_data = json.loads(response.content)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(len(response_data['users']), 2)
+        self.assertContains(response, 'test2@mail.com')
```
Comment (lukesa) [inline teams/tests/tests_list_user_by_team.py]: get value from govuser1 object

File: applications/views.py [hunk 6]
```diff
@@ -123,7 +124,10 @@ def post(self, request):
 
 
 class ApplicationDetail(APIView):
-
+    authentication_classes = (GovAuthentication,)
```
Comment (cstobart) [inline applications/views.py]: doctoring above auth method

File: audit/views.py [hunk 7]
```diff
@@ -0,0 +1,26 @@
+import json
+
+from django.http import JsonResponse
+from rest_framework.views import APIView
+from reversion.models import Version, Revision
+
+
+class AuditDetail(APIView):
+    """
```
Comment (cstobart) [inline audit/views.py]: put gov auth on this end point

**Pull request merged**